airbyte-protocol/models/src/main/java/io/airbyte/protocol/models/Field.java

```diff
@@ -55,4 +57,21 @@ public String getTypeAsJsonSchemaString() {
     return type.name().toLowerCase();
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
can we use equals builder for easier extensibility?

Reply:
this is still using the intellij template. happy to use a different template if you have one you want to share.